glaciers.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Dec 11 08:15:21 2018
+
+@author: courtney
+"""
+
+import subprocess
+import os
+import matplotlib.pyplot as plt
+
+init_path = r"/home/courtney/Desktop/practice/"
```

Comment:
What is this path supposed to be for? i.e. if I were to use this code myself, how would I know what my equivalent of this path would be?

Reply:
Good point, should I write a comment clarifying what folders and files should be present in that initial directory? How can I make this more transparent?